tests/test_all_rules.py

```diff
@@ -292,6 +292,48 @@ def test_required_tags(self):
             if missing_required_tags or is_missing_any_tags:
                 self.fail(error_msg)
 
+    def test_primary_tactic_as_tag(self):
+        from detection_rules.attack import tactics
+
+        invalid = []
+        tactics = set(tactics)
+
+        for rule in self.rules:
+            rule_tags = rule.contents['tags']
+
+            if 'Continuous Monitoring' in rule_tags or rule.type == 'machine_learning':
```

Comment:
why do these get  special treatment?

Reply:
Most of the continuous monitoring rules are not inherently threat-centric rules, though they can certainly be mapped to a threat (in some instances). So early on, it was decided to leave it out of the tags. If we feel that has changed, we can update.

Am I missing anything @bm11100 ? I forget the context to ML